### PR TITLE
Updatedcontact

### PR DESCRIPTION
**Pull request by ptabhishekutopia**
image
improving image path

### CODE REVIEW

**Comment by Arioum** [comment]
Try not to use `<center/>` tag. It works, but it is deprecated and not encouraged.

File: src/components/Contact.tsx [hunk 1]
```diff
@@ -42,82 +39,75 @@ export default function Contact(props: ContactType) {
...
+                    </SimpleGrid>
+                    </center>
+                    </Box>
+             
+                  
+               
+                <Box boxShadow='md' p='6' rounded='md' bg='white'  borderRadius="lg"  w={"100%"} margin={"5px 0px"}>
+                 
                       <form name="contact" method="post" data-netlify-honeypot="bot-field" data-netlify="true">
+                      <Box py={{ base: 5, sm: 5, md: 8, lg: 10 }}w={"100%"} >
```
Comment (Arioum) [inline src/components/Contact.tsx]: There appears to be no space between `py `and `w `attribute. Is the `w` still working as expected?

File: src/components/Contact.tsx [hunk 2]
```diff
@@ -42,82 +39,75 @@ export default function Contact(props: ContactType) {
...
-                        <MdPhone color="#1970F1" size="20px" />{" "}
-                        {props?.phoneNumber}
-                      </Link>
+                </Box>
 
-                      <Link href={"mailto:" + props?.email} isExternal display={'fled'} gap={'8px'} alignItems={'center'}>
-                        <MdEmail color="#1970F1" size="20px" />{" "}
-                        {props?.email}
+                <Box py={{ base: 5, lg: 10 }} w={"100%"} bg={"#edf2f7"}>
+                <center>
```
Comment (saurabh73) [inline src/components/Contact.tsx]: @fThAbhishek-Pandey Remove this

**Comment by ptabhishekutopia** [comment]
> Try not to use `<center/>` tag. It works, but it is deprecated and not encouraged.

I will trying to  fixed this email

**Comment by Arioum** [comment]
2 minor things you can improve:

**1. Centering of the below elements when the screen size is less:**
[](image.png)



**2. Try giving space between the contact form and the footer:**
[](image.png)

**Comment by ptabhishekutopia** [comment]
> 2 minor things you can improve:
> 
> **1. Centering of the below elements when the screen size is less:** [](image.png)
> 
> **2. Try giving space between the contact form and the footer:** [](image.png)

I think content of address is more than other filled . so, the icon is shown large comparatibily

File: src/components/Contact.tsx [hunk 3]
```diff
@@ -29,144 +28,186 @@ import {
...
 
 export type ContactType = NonNullable<IndexPageFrontmatterType>["contact"];
 
 export default function Contact(props: ContactType) {
-
   return (
     <Container
       id="contact"
-      maxW={'6xl'}
+      maxW={"100%"}
```
Comment (saurabh73) [inline src/components/Contact.tsx]: use `'6xl'`

File: src/components/Contact.tsx [hunk 4]
```diff
@@ -29,144 +28,186 @@ import {
...
+      
+          <Flex w={"100%"} justifyContent={"space-between"}>
+            {/* left box */}
 
-                      <Link href={"mailto:" + props?.email} isExternal display={'fled'} gap={'8px'} alignItems={'center'}>
-                        <MdEmail color="#1970F1" size="20px" />{" "}
-                        {props?.email}
-                      </Link>
+            <Box
+              w={"40%"}
```
Comment (saurabh73) [inline src/components/Contact.tsx]: give responsive width

File: src/components/Contact.tsx [hunk 5]
```diff
@@ -29,144 +28,186 @@ import {
...
+              boxShadow="md"
+              m={"0px auto 3rem auto"}
+              p="6"
```
Comment (saurabh73) [inline src/components/Contact.tsx]: responsive padding values.

File: src/components/Contact.tsx [hunk 6]
```diff
@@ -29,144 +28,186 @@ import {
...
+                        background : "red.500"
+                      }}
+                    >
+                      Send Message
+                    </Button>
+                  </Flex>
+                </FormControl>
+              </form>
+            </Box>
+            <Box w={"50%"}>
```
Comment (saurabh73) [inline src/components/Contact.tsx]: Use responsive width

File: src/components/Footer.tsx [hunk 7]
```diff
@@ -51,25 +51,53 @@ export default function Footer(props: FooterPropType) {
...
             </Stack>
-            <Stack direction={"column"} spacing={4} align={"flex-start"}>
+            <Stack direction={"column"} spacing={1} align={"flex-start"}>
               <SocialButton label={"Twitter"} href={"#"}>
-                <FaTwitter size={"2rem"} />
+                <FaTwitter size={"3rem"} />
               </SocialButton>
               <SocialButton label={"YouTube"} href={"#"}>
-                <FaYoutube size={"2rem"} />
+                <FaYoutube size={"3rem"} color="red.400" />
```
Comment (saurabh73) [inline src/components/Footer.tsx]: Should be Linkedin inon

File: src/components/Footer.tsx [hunk 8]
```diff
@@ -51,25 +51,53 @@ export default function Footer(props: FooterPropType) {
...
+                  href={"#"}  
+                  _hover={{
+                    textDecoration: "none",
+                  }} >Privacy Policy</Link>
             </Stack>
-            <Stack direction={"column"} spacing={4} align={"flex-start"}>
+            <Stack direction={"column"} spacing={1} align={"flex-start"}>
               <SocialButton label={"Twitter"} href={"#"}>
-                <FaTwitter size={"2rem"} />
+                <FaTwitter size={"3rem"} />
```
Comment (saurabh73) [inline src/components/Footer.tsx]: Should query from Decap

File: src/data/index.md [hunk 9]
```diff
@@ -70,10 +70,39 @@ testimonials:
...
 blurbs:
   blogList:
     - blurb: "Conversational Revolution: Unleashing the Power of AI Chatbots"
+footer:
+  social:
+    instagram:
+      logo:
+        CoLor: "#ffffff"
+        BGCoLor: "#f56565"
+        icon: FiInstagram
```
Comment (saurabh73) [inline src/data/index.md]: LinkedIn, Twitter and Github Icon

File: static/admin/config.yml [hunk 10]
```diff
@@ -2,69 +2,67 @@ backend:
...
+      - file: 'src/data/index.md'
+        label: 'Landing Page'
+        name: 'index'
         fields:
           - {
-              label: "Template Key",
-              name: "templateKey",
-              widget: "hidden",
-              default: "index-page",
+              label: 'Template Key',
```
Comment (saurabh73) [inline static/admin/config.yml:55]: Unnecessary change.

File: static/admin/config.yml [hunk 11]
```diff
@@ -109,7 +107,12 @@ collections:
                           widget: object,
                           fields:
                             [
-                              { label: Icon, name: icon, widget: icon, iconFamily: di },
+                              {
```
Comment (saurabh73) [inline static/admin/config.yml:109]: Unnecessary change.

**Review by saurabh73 — CHANGES_REQUESTED** [review]

**Comment by ptabhishekutopia** [comment]
resolved

**Review by Arioum — COMMENTED** [review]
### Can you fix this? 
Currently the form goes outside the viewport before being wrapped.
<br/>
**You can do this by:**
* Reducing the width of both image and form once they have no space. You can reset this width once it has wrapped. So keep the width responsive rather than fixed `px` value;

https://github.com/user-attachments/assets/6d1b9e62-7ad6-4217-b9f3-b8eba5c33c3d

**Comment by ptabhishekutopia** [comment]
> ### Can you fix this?
> Currently the form goes outside the viewport before being wrapped. **You can do this by:**
> 
> * Reducing the width of both image and form once they have no space. You can reset this width once it has wrapped. So keep the width responsive rather than fixed `px` value;
> 
>  mq-issue-outofbounds.mp4


Uploading Recording 2024-07-19 194428.mp4…

updated

**Review by saurabh73 — APPROVED** [review]

**Pull request merged**